### DISCUSSION TITLE
Unsubscribe links for notifications

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1275] [Feature Requests]
> I'm really looking to use your plugin as a means to notify paying subscribers that new content has been published. I love the plugin and the way its setup, but users will need to be able to disable the notifications if they choose, preferably through a front end or email link click

That sounds like a useful new feature we could add. Perhaps it's a shortcode that people can include in the notifications.

**Comment by stevejburge** [comment]
One more on this:

> I've not found a way for users to be able to elect NOT to receive emails for content. I know there is the "notify me" feature, but I can't find a way to make this accessible on the front end, nor for it to apply to certain post types.

**Comment by stevejburge** [comment]
One more request for this today

**Comment by stevejburge** [comment]
One more request for this

**Comment by stevejburge** [comment]
> is it possible to include to mail template for notification some kind of link for unsubscribe?

**Comment by stevejburge** [comment]
One more request. This user would like to have the subscribe / unsubscribe options on the frontend so they can add it to user profiles:

> How to put the Editorial Notifications workflow Channel in the Frontend Profile User? I'm using Ultimate Members plugin for the Frontend Profile. I would like our members to select "receive email" or "muted" in Frontend.

**Comment by stevejburge** [comment]
One more:

> Is there a way to have an unsubscribe link in the Notify when content is published?

**Comment by angeladesouza** [comment]
Love this plugin - it's really solved a problem for me. But I fear getting complaints as there's no way to unsubscribe. 
I would very much like to have a quick unsubscribe option. Any ideas when this can get added? A shortcode perhaps or a link? Is there any workaround at the moment?

**Comment by stevejburge** [comment]
@angeladesouza Sorry, not yet, but we're planning to add this

**Comment by stevejburge** [comment]
> The most important issue is perhaps if "users" has to login to the site from the notification link, and if they can Unsubscribe with a simple click?

**Comment by stevejburge** [comment]
> How do you mange opt out option? And can I use any sign-up plugin ? I have the notification working for me as admin but don't want to spam people so I need them to signup in a box somewhere.
